spec/fieldhand/record_spec.rb

```diff
@@ -35,6 +35,39 @@ module Fieldhand
       end
     end
 
+    describe '#to_xml' do
+      it 'returns the whole element even if there is no metadata as a string' do
+        element = ::Ox.parse('<record/>')
+        record = described_class.new(element)
+
+        expect(record.to_xml).to eq("\n<record/>\n")
+      end
+
+      it 'returns the whole element as a string' do
+        element = ::Ox.parse("<record><metadata>Foo</metadata></record>")
+        record = described_class.new(element)
+
+        expect(record.to_xml).to eq(<<-XML)
+
+<record>
+  <metadata>Foo</metadata>
+</record>
+        XML
+      end
+
+      it 'returns the whole element with unicode characters as a string' do
+        element = ::Ox.parse("<record><metadata>ψFooϨ</metadata></record>")
+        record = described_class.new(element)
+
+        expect(record.to_xml).to eq(<<-XML)
+
+<record>
+  <metadata>ψFooϨ</metadata>
```

Comment:
For Ruby 1.8 support, you may need to add a magic comment to the top of this file:

```ruby
# encoding: utf-8
```